tests/super_test.py

```diff
@@ -110,6 +112,19 @@ def test_fix_super_noop(s):
             '    def f(cls):\n'
             '        super().f()\n',
         ),
+        pytest.param(
+            'class C:\n'
+            '    async def foo(self):\n'
+            '        super(C, self).foo()\n',
+            'class C:\n'
+            '    async def foo(self):\n'
+            '        super().foo()\n',
+            id='super inside async func def',
+            marks=pytest.mark.xfail(
+                sys.version_info < (3, 5),
+                reason='async introduced in python 3.5',
+            ),
+        ),
```

Comment:
let's split this into a separate function

Reply:
Addressed in 626fead0f4bba195eed39bc5ee9b1b9612db4bfc.